### ISSUE TITLE
Failing testing, `test_psdp_woodgate_non_full_rank`

### ISSUE THREAD

**Issue by FanwangM** [theochem/procrustes#190]
Can you check why the testing is failing for https://github.com/theochem/procrustes/runs/8172507556?check_suite_focus=true? I have tried to rerun the testing and it didn't help. I don't know if it's `Scipy` or `Numpy` version related or operating system related. The tests work before we merge the PR.

Thank you. @banrovegrie

**Comment by banrovegrie** [comment]
Yeah I will fix this

**Comment by banrovegrie** [comment]
It's weird how the issue doesn't exist at all for python3.8+ as well as python3.6-.

**Comment by FanwangM** [comment]
> It's weird how the issue doesn't exist at all for python3.8+ as well as python3.6-.

True. I think it's Windows OS related only as I have tested for many times.

**Comment by FanwangM** [comment]
Just figured that the failing test was running on Windows 32 machine, which is a big surprise to me.

**Comment by FanwangM** [comment]
I have tested it again on a windows machine with the same working environment (Python 3.7.9 and the same numpy and scipy version) and it works. Also, the current running of tests, https://github.com/theochem/procrustes/runs/8197877947?check_suite_focus=true, works. So, I would guess that's related to the hardware? 

What do you think? 
@PaulWAyers @banrovegrie

**Comment by PaulWAyers** [comment]
Seems strange, but not implausible, to me. I guess we can specify the hardware in the install?

**Comment by banrovegrie** [comment]
Maybe, but will that lead to a precision issue as big as 3 decimal places?

**Comment by banrovegrie** [comment]
Let's have a meet and fix the currently present testing issues. Will anytime on the 10th of October (or later) work?

**Comment by FanwangM** [comment]
What's the time difference between us now (your current location and Toronto)?

**Comment by banrovegrie** [comment]
I got back to college yesterday. So now the time gap is around 9 hours. But I will mostly be available till late at night (India time) which should be around 3:30 PM Toronto time.

**Comment by banrovegrie** [comment]
@FanwangM any updates?
